contracts/Voter.sol

```diff
@@ -356,7 +356,7 @@ contract Voter is IVoter, ERC2771Context, ReentrancyGuard {
   function claimRewards(address[] memory _gauges) external {
     uint256 _length = _gauges.length;
     for (uint256 i = 0; i < _length; i++) {
-      IGauge(_gauges[i]).claimReward(_msgSender());
+      IRewardGauge(_gauges[i]).claimReward(_msgSender());
```

Comment:
shall we move this function to a helper contract?
1. we can delete the restriction in `claimReward`, which only allowed account owner and voter to claim reward
2. keep voter contract simple